mesheryctl/cmd/perf.go

```diff
@@ -79,7 +79,7 @@ var perfCmd = &cobra.Command{
 		if len(testURL) > 0 {
 			postData = postData + "\nendpoint_url: " + testURL
 		} else {
-			println("Error: Please enter a TestURL")
+			println("Error: Please enter a TestURL \nExample: http://my-service/api/v1/test")
```

Comment:
@gunjanmimo I'm not sure that you understand the issue. Let's chat through it on tomorrow's call. @kanishkarj or @anandrkskd can offer some guidance.

Reply:
@gunjanmimo any progress to report here?